### ISSUE TITLE
[ISSUE] Slave Server can't connect to master on non-standard HTTPS port

### ISSUE THREAD

**Issue by weirdcrap** [WillyXJ/facileManager#382]
fM Version : **3.0.0**
fmDNS Version : **3.0.0**

**In raising this issue, I confirm the following (please check boxes, eg [X]):**

- [X] I have read and understood the [contributors guide](https://github.com/WillyXJ/facileManager/blob/master/.github/CONTRIBUTING.md).
- [X] I have checked that the bug-fix I am reporting can be replicated, or that the feature I am suggesting isn't already present.
- [X] I have checked that the issue I'm posting isn't already reported.
- [X] I have checked that the issue I'm posting isn't already solved and no duplicates exist in [closed issues](https://github.com/WillyXJ/facileManager/issues?q=is%3Aissue+is%3Aclosed) and [opened issues](https://github.com/WillyXJ/facileManager/issues)
- [x] I have checked the pull requests tab for existing solutions/implementations to my issue/suggestion.

---

**(ISSUE) Expected Behavior:**
The server should be able to connect and pull changes from the master server regardless of whether I use a standard or non-standard HTTPS port.

**(ISSUE) Actual Behavior:**
I'm not sure when this changed but at some point in the past year my slave server has been unable to connect to the master server to pull zone changes and I just now discovered it as we almost never make changes anymore in this panel.

When I run the cron tab by hand or when I try to perform an upgrade/re-install of fmDNS I get an error:

```
sudo /usr/bin/php /usr/local/facileManager/fmDNS/client.php cron
fm.example.com:44433 is currently not available via tcp/443.  Aborting.


sudo php /usr/local/facileManager/fmDNS/client.php upgrade
fm.example.com:44433 is currently not available via tcp/443.  Aborting.

sudo php /usr/local/facileManager/fmDNS/client.php re-install
fm.example.com:44433 is currently not available via tcp/443.  Aborting.
```

The master server is using a self-signed certificate with a non-standard HTTPS port like so: https://fm.example.com:44433

It looks like the values are hard coded in the functions.php file? Is there anyway we could make this a variable?

I can manually query the server with `curl -k https://fm.example.com:44433` without any issues.

I don't have another server to test on but it seems like if you try to use a non-standard HTTPS (probably HTTP as well) port fm doesn't know how to handle it and just throws a can't connect error.

Everything works fine on the master server with fmServer on it, only the slave is affected.

**(ISSUE) Steps to reproduce:**
Setup two servers and have the master server use a non-standard HTTPS port. The slave server will be unable to connect to it due to it's non-standard port.

**Comment by weirdcrap** [comment]
Once I moved the web interface to the standard 443 port, everything is working fine, so this is just a non-standard port issue.

**Comment by WillyXJ** [comment]
Thanks for the report.  Indeed this has changed by mistake.  I'll work on a fix for 3.0.1.

**Comment by WillyXJ** [comment]
This is now fixed in [3.0.1](http://www.facilemanager.com/download/) and later.
